### DISCUSSION TITLE
Extend IBM Plex to Bengali Typeface Family.

### DISCUSSION THREAD

**Discussion by mistu01** [IBM/plex#471] [Ideas]
**Thank you** so much for everything. 

As you guys already did _Devanagari_ I want to suggest to extend the family and add _Bengali_ typefaces. Bengali is a native language for more than 300 million people all around the world, also more than 37 million people use it as their 2nd language. Sadly there are not much Good free/Open Source fonts available for this Beautiful language. As a native speaker, I would love to see IBM Plex has a _Bengali_ family. 

Kind Regards. ^_^

**Comment by tay1orjones** [comment]
cc @BoldMonday

**Reply by BoldMonday** [reply]
Added as issue #478
